src/components/Comment.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+    Box,
+    Button,
+    Container,
+    Grid,
+    Typography
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+
+    root: {
+        borderStyle: 'solid'
+    },
+    date: {
+        float: 'left',
+        textAlign: 'left',
+        fontSize: '18px',
+        marginTop: '0.25rem',
+    },
+    content: {
+        marginTop: '0.75rem'
+    },
+    deleteComment: {
+        paddingLeft: '2rem',
+        paddingRight: '2rem',
+        float: 'right',
+        textAlign: 'right',
+        background: 'lightgrey'
+    }
+
+});
+
+interface CommentProps {
+    username: string;
+    date: string;
+    claps: number;
+    content: string;
+}
+
+const Comment: React.FC<CommentProps> = ({username, date, claps, content}) => {
```

Comment:
I know this will be kind of annoying but maybe don't destruct the props. It'll be easier to understand if we just call the argument `props` and call `prop.username` and similar when needed